src/api/app/models/relationship.rb

```diff
@@ -158,6 +166,20 @@ def check_global_role
   def allowed_user
     raise NotFoundError, "Couldn't find user #{user.login}" if user && user.is_nobody?
   end
+
+  def event_create
+    return if User.session == user
```

Comment:
Not sure we should skip this, this could rather be limited by `receiver_roles` right?

Reply:
True. `receiver_roles` could be set to define who gets notified. But it is not need for now. We can analyze `receiver_roles` better when we implement the notifications.